ops/cli.py

```diff
@@ -1,351 +1,343 @@
+        info("=>  Running cloud_sql_proxy")
+        token = shell("gcloud auth print-access-token")
+        connectionName = json.loads(shell(f"gcloud --project {self.project} --format=json sql instances"
+                                          f"describe {self.cloudsql_name}"))["connectionName"]
+        return shell(f"docker run --rm -d -p 127.0.0.1:5432:5432 gcr.io/cloudsql-docker/gce-proxy:1.16 "
+                     f"/cloud_sql_proxy -token='{token}' -instances='{connectionName}=tcp:0.0.0.0:5432'")
+
 
-        commands_available()
-        set_up_helm()
-        set_up_k8s(project=project, namespace=args.namespace, cluster=cluster, zone=args.zone)
+class DeployCommand(ConnectCommand):
+    """Class for deploy command that deploys WFL to infrastructure."""
 
-        version = shell("cat version")
-        publish_docker_images(version=version)
+    def __init__(self, parsed_args: argparse.Namespace):
+        super().__init__(parsed_args)
+
+    def execute(self) -> int:
+        """Set up helm, publish docker, render CTMPL, helm deploy, migrate liquibase."""
+
+        if not input(f"Are you sure you want to deploy version {self.version}? [N/y]").lower().startswith("y"):
+            exit(0)
+
+        self.__set_up_helm()
+
+        self.__publish_docker_images()
 
         deploy = os.path.join("derived", "helm", "deploy")
         if not os.path.exists(deploy):
             os.makedirs(deploy)
 
         values = "wfl-values.yaml"
-        ctmpl = f"derived/2p/gotc-deploy/deploy/{gotc_folder}/helm/{values}.ctmpl"
+        ctmpl = f"derived/2p/{self.gotc_deploy_folder}/helm/{values}.ctmpl"
         shutil.copy(ctmpl, deploy)
 
-        render_ctmpl(ctmpl_file=f"{deploy}/{values}.ctmpl", WFL_VERSION=version)
-        helm_deploy_wfl(values=f"{deploy}/{values}")
+        render_ctmpl(ctmpl_file=f"{deploy}/{values}.ctmpl", WFL_VERSION=self.version)
+        self.__helm_deploy_wfl(values=f"{deploy}/{values}")
 
-        container = run_cloudsql_proxy(project=project, cloudsql_instance_name="zero-postgresql")
-        with open(f"{deploy}/{values}") as f:
-            helm_values = yaml.safe_load(f)
+        container = self.run_cloudsql_proxy()
+        with open(f"{deploy}/{values}") as values_file:
+            helm_values = yaml.safe_load(values_file)
             env = helm_values['api']['env']
-            run_liquibase_migration(env['ZERO_POSTGRES_USERNAME'], env['ZERO_POSTGRES_PASSWORD'])
-            info("=> Stopping cloud_sql_proxy")
-            shell(f"docker stop {container}")
+            self.__run_liquibase_migration(env['ZERO_POSTGRES_USERNAME'], env['ZERO_POSTGRES_PASSWORD'])
+        info("=>  Stopping cloud_sql_proxy")
+        shell(f"docker stop {container}")
+
+        success("Deployment is done!")
+        info(shell("kubectl get pods"), plain=True)
 
-        success("[✔] Deployment is done!")
-        shell("kubectl get pods")
         return 0
 
+    @staticmethod
+    def __set_up_helm():
```

Comment:
Why is this a static method? What's the benefit of this class owning this behaviour rather than just making it a free function?

Reply:
1. It is private for better encapsulation (no need to pollute the namespace)
2. Editors can "fold" easier by this existing inside a class, making it quicker to get where you need in the file

Reply:
> It is private for better encapsulation (no need to pollute the namespace)
That's not what encapsulation is. Encapsulation is the union of data and operations so that one manipulates the essence of the data without exposing the data itself. There's no encapsulation here as none of `self`'s properties are used.
Why is polluting the global namespace here such an issue?

> Editors can "fold" easier
I'm using VSCode which handles functions just fine? If anything this funciton is one that shouldn't be folded - everything else is just boilerplate.

Reply:
Re folding I mean that editors can fold classes down, just another layer of organization that some folks still might not use.
Polluting the namespace might not be a big deal for us, I dunno. I see it as a code quality thing but maybe that's just my out-of-touch academia talking.

Reply:
Resolved in new version, no methods exist in the file.